docs/tutorials/initial-data.md

```diff
@@ -74,284 +74,3 @@ module.exports = {
 ```
 
 > **Tip:** A similar setup can be achieved by running the Keystone CLI `yarn create keystone-app` and selecting `Starter (Users + Authentication)`. This starter project has a `User` list, `PasswordAuthStrategy` and seeding of the database already configured. For now, we will proceed manually.
-
```

Comment:
Removing this documentation has rendered this document useless. We should replace the old content with new content which refers to the `server-side-graphql-client` package. It needn't be as verbose as this content was, since we aren't describing a new APi, but this tutorial should continue to serve a purpose.